AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
Parceler can also handle boxed primitives in collections:
`List<Integer>`, `Map<Integer, String>`
A mix:
`Map<String, AnnotatedWithParcel>`
And nested collections:
`Map<List<String>, List<AnnotatedWithParcel>>`
Would you like these cases handled as well?